frontend/src/theme/utils.ts

```diff
@@ -427,20 +427,59 @@ export const getSystemTheme = (): ThemeConfig => {
     : lightTheme
 }
 
+// Method taken from
+// https://stackoverflow.com/questions/16427636/check-if-localstorage-is-available
+export const localStorageAvailable = (): boolean => {
+  const { localStorage } = window
```

Comment:
Looks like the error still exists in incognito. I wonder if just accessing `localStorage` from window should be in the try/catch block.

Reply:
:+1: that was it. Thanks!